moveit_ros/planning/rdf_loader/CMakeLists.txt

```diff
@@ -1,8 +1,15 @@
 set(MOVEIT_LIB_NAME moveit_rdf_loader)
 
-add_library(${MOVEIT_LIB_NAME} src/rdf_loader.cpp)
+add_library(${MOVEIT_LIB_NAME} SHARED src/rdf_loader.cpp)
 set_target_properties(${MOVEIT_LIB_NAME} PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${MOVEIT_LIB_NAME} ${catkin_LIBRARIES})
+ament_target_dependencies(${MOVEIT_LIB_NAME}
+      rclcpp
+      ament_index_cpp
+      urdf)
```

Comment:
In the master branch this's replaced with 

```
target_link_libraries(${MOVEIT_LIB_NAME}
  ${Boost_LIBRARIES}
  ${srdfdom_LIBRARIES}
  ${urdf_LIBRARIES}
  ${rclcpp_LIBRARIES}
  ${pluginlib_LIBRARIES}
  ${moveit_core_LIBRARIES}
)

```

Reply:
I tried to compile it without changing it to `target_link_libraries` and it doesn't fail